ui/css/itcss/components/network.scss

```diff
@@ -1,195 +0,0 @@
-@use "design-system";
-
-.network-component--disabled {
```

Comment:
Not used

![Screenshot 2025-02-07 at 11 42 59 AM](https://github.com/user-attachments/assets/aabb078a-de84-42ec-b6ca-4af69d0feab9)